pkg/service/id_local.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"fmt"
+
+	"github.com/cilium/cilium/common"
+	"github.com/cilium/cilium/common/types"
+	"github.com/cilium/cilium/pkg/lock"
+)
+
+var (
+	mutex      lock.RWMutex
+	servicesID = map[uint32]*types.L3n4AddrID{}
```

Comment:
Please comment what are the keys and values for those maps.

Reply:
fixed